docs/src/content/docs/ru/challenges/performance/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Производительность Angular-а
+prev: false
+next: false
+contributors:
+  - tomalaforge
+  - tomer953
+description: Узнайте, как использовать расширение Angular DevTools для Chrome.
+noCommentSection: true
+sidebar:
+  order: 1
+---
+
+import { LinkCard } from '@astrojs/starlight/components';
+
+В этой серии испытаний по производительности вы узнаете, как оптимизировать и улучшить производительность вашего приложения Angular.
+
+Прежде чем приступить к решению какого-либо испытания, я приглашаю вас скачать [расширение Angular DevTools для Chrome](https://chrome.google.com/webstore/detail/angular-devtools/ienfalfjdbdpebioblfackkekamfmbnh), если вы еще этого не сделали.
+
+Это расширение позволяет профилировать ваше приложение и обнаруживать проблемы производительности, что очень полезно для понимания мест, где могут возникать проблемы с производительностью.
+
+## Как его использовать
+
+При запуске приложения Angular вы можете проверить страницу, нажав <b>F12</b>, что откроет <b>Инструменты разработчика Chrome</b>. Затем перейдите на вкладку <b>Angular</b>. Оттуда вы можете выбрать вкладку <b>Profiler</b>, как показано ниже.
+
+![вкладка профилировщика](../../../../assets/performance/profiler-tab.png 'Вкладка профилировщика')
```

Comment:
the build is failing because of this path.

Because submitting your work, try to run `npm run doc:dev` in your local env to see if the project starts. 👍

Reply:
Hi, I tried to launch it, everything was ok, but sorry yes I didn't check this page